### PR TITLE
studio: show a message or error component when there are no claimants #669

### PR DESCRIPTION
**Pull request by ishpaul777**
https://github.com/factly/dega/issues/669

### CODE REVIEW

File: studio/src/pages/claims/index.js [hunk 1]
```diff
@@ -86,6 +116,8 @@ function Claims({ permission }) {
     return { claims: [], total: 0, loading: state.claims.loading };
   });
 
+  console.log(ratingsCount, loading)
```
Comment (surajmn1) [inline studio/src/pages/claims/index.js]: @ishpaul777 remove logs

File: studio/src/pages/claims/index.js [hunk 2]
```diff
@@ -115,7 +147,34 @@ function Claims({ permission }) {
...
+    title={'Please create a claimant.'}
+    subTitle="You cannot create a claim without a claimant."
+    extra={
+      <Link to="/claimants/create">
+        <Button type="primary">Create claimant</Button>
+      </Link>
+    }
+  />)
+  }
+
```
Comment (surajmn1) [inline studio/src/pages/claims/index.js]: @ishpaul777 can the case when both ratings and claimants aren't present be handled?

Comment (ishpaul777) [reply studio/src/pages/claims/index.js]: you mean showing both at same time ?

Comment (surajmn1) [reply studio/src/pages/claims/index.js]: we can refactor this to have create rating and create claimants button inside and then conditionally render both or one.

**Review by surajmn1 — DISMISSED** [review]

**Review by surajmn1 — DISMISSED** [review]

**Review by surajmn1 — DISMISSED** [review]

**Review by surajmn1 — APPROVED** [review]

**Comment by surajmn1** [comment]
@shreeharsha-factly please merge.

**Pull request merged**